README.md

```diff
@@ -4,9 +4,13 @@
 
 # ddev
 
-The purpose of *ddev* is to support developers with a local copy of a site for development purposes. It runs the site in Docker containers.
+The purpose of *ddev* is to help Drupal and WordPress abstract away the complexity and effort of adopting and maintaining a docker based local development stack. You can see all "ddev" usages using the help commands, like ddev -h, ddev start -h, etc.
```

Comment:
s/help Drupal and WordPress abstract/help Drupal and WordPress developers abstract/